Minio/ApiEndpoints/IObjectOperations.cs

```diff
@@ -54,8 +54,8 @@ public interface IObjectOperations
         /// <param name="size">Size of stream</param>
         /// <param name="contentType">Content type of the new object, null defaults to "application/octet-stream"</param>
         /// <param name="cancellationToken">Optional cancellation token to cancel the operation</param>
-
-        Task PutObjectAsync(string bucketName, string objectName, Stream data, long size, string contentType = null, CancellationToken cancellationToken = default(CancellationToken));
+        /// <param name="metaData">Object metadata to be stored. Defaults to null.</param>
```

Comment:
parameter description should indicate that `metaData` is also an optional parameter.